### ISSUE TITLE
Error with timezone not UTC ?

### ISSUE THREAD

**Issue by edelamusse** [binarypatrick/Prune#10]
- OS: Linux Debian 12 bookworm
- Prune installed with the following commands (see: https://dev.to/binarypatrick/using-prune-to-manage-backups-3nfg):
```
curl -s -o prune-install-linux-x64.sh https://raw.githubusercontent.com/BinaryPatrick/Prune/main/scripts/install-linux-x64.sh
chmod +x prune-install-linux-x64.sh
sudo ./prune-install-linux-x64.sh
rm prune-install-linux-x64.sh
```
- Command and error:
```
$ prune --path backup/xxx/ --keep-last 5 --dry-run --verbose
Searching backup/xxx/
3 files found matching *
The UTC time represented when the offset is applied must be between year 0 and 10,000. (Parameter 'offset')
System.ArgumentOutOfRangeException: The UTC time represented when the offset is applied must be between year 0 and 10,000. (Parameter 'offset')
   at System.DateTimeOffset.ValidateDate(DateTime dateTime, TimeSpan offset)
   at System.DateTimeOffset..ctor(DateTime dateTime)
   at System.DateTimeOffset.op_Implicit(DateTime dateTime)
   at BinaryPatrick.Prune.Models.RetentionSorter..ctor(IConsoleLogger logger, IEnumerable`1 files)
   at BinaryPatrick.Prune.Services.RetentionSorterFactory.CreateRetentionSorter(IEnumerable`1 files)
   at BinaryPatrick.Prune.Services.PruneService.PruneFiles()
   at BinaryPatrick.Prune.Program.<>c.<Main>b__0_0(PruneOptions options)
```

Normal ?

I specify that my timezone is not UTC but Europe/Paris (UTC+2):

```
$ timedatectl 
               Local time: Sun 2024-05-05 18:07:13 CEST
           Universal time: Sun 2024-05-05 16:07:13 UTC
                 RTC time: Sun 2024-05-05 16:07:13
                Time zone: Europe/Paris (CEST, +0200)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: no
```

**Comment by binarypatrick** [comment]
Hey, I missed this but I am investigating now.

**Comment by binarypatrick** [comment]
I was able to fix this with [v1.0.3](https://github.com/binarypatrick/Prune/releases/tag/v1.0.3)
